### PR TITLE
CFP - trim down the details required

### PR DESCRIPTION
**Pull request by PeterGrand**
My suggested edits to simplify the CFP requirements. Big thing is removing cost estimates and reducing the detail on catering. If the catering can support the main conference days, the rest will work out.

There's no way to get a good A/V quote for a CFP, I think.

### CODE REVIEW

File: _posts/2023-06-17-call-for-venue-proposal-2024.md [hunk 1]
```diff
@@ -29,32 +29,23 @@ We need some information about your city, about the venue, the hotel, and travel
...
-		* Main conference, 3 days—450 people per day: Breakfast, morning break, lunch, afternoon break, and all day beverage service. (All day beverage service includes water stations, coffee, assorted black and herbal teas, and a variety of soft drinks.)
-		* Sprints, 2 days—75 people each day: Breakfast, morning break, lunch, afternoon break, and all day beverage service. (All day beverage service includes water stations, coffee, assorted black and herbal teas, and a variety of soft drinks.)
+	* Explanation of catering options available at the venue that can cover:
+		* 450 people per day: Breakfast, morning break, lunch, afternoon break, and all day beverage service.
 		* Catering must have options for celiacs, vegetarians, and vegans of equal quality and variety as main menu. Estimate 15% vegetarian, 5% vegan, and 5-10% gluten-free (some of which may overlap with the other two groups).
-
-	* Costs for:
-		* Function space
-		* Catering
-		* Audio-visual services
```
Comment (jefftriplett) [inline _posts/2023-06-17-call-for-venue-proposal-2024.md:51]: @adamfast should we pull this or reword it?

File: _posts/2023-06-17-call-for-venue-proposal-2024.md [hunk 2]
```diff
@@ -70,15 +61,14 @@ We need some information about your city, about the venue, the hotel, and travel
 	* Location
 	* Transportation options (how do people get there from the hotel?)
 	* Attendance capacity
-	* Cost of function space, catering, and audio-visual services. Sprints are often hosted by local companies as sponsorship of the event, which is preferable, as it saves the cost of renting a venue.
+	* Information on local companies that might host the sprints at no cost.
```
Comment (jefftriplett) [inline _posts/2023-06-17-call-for-venue-proposal-2024.md]: Might be worth a note and I'm not in-love with this wording "if sprints aren't economically feasible to host as part of the same venue..."

File: _posts/2023-06-17-call-for-venue-proposal-2024.md [hunk 3]
```diff
@@ -29,32 +29,23 @@ We need some information about your city, about the venue, the hotel, and travel
 	* Location
 	* Transportation options (how do people get there from the airport?)
 	* Attendance capacity
-	* Layout of function space:
+	* Assurance that the venue can provide the following spaces (A floorplan with explanation of what will go where would be great.):
```
Comment (drewbrew) [inline _posts/2023-06-17-call-for-venue-proposal-2024.md]:
```suggestion
	* Assurance that the venue can provide the following spaces (A floorplan with a rough explanation of what will go where would be great.):
```

File: _posts/2023-06-17-call-for-venue-proposal-2024.md (hunk 2 above)
Comment (drewbrew) [reply _posts/2023-06-17-call-for-venue-proposal-2024.md]:
```suggestion
	* Information on local companies that might be willing to host the sprints at no cost.
```

File: _posts/2023-06-17-call-for-venue-proposal-2024.md [hunk 4]
```diff
@@ -29,32 +29,23 @@ We need some information about your city, about the venue, the hotel, and travel
...
+		* Space for supplemental beverage stations for daylong service.
 		* Secure overnight storage for vendor booth supplies and conference registration desk items.
 
-	* Catering:
-		* Tutorials day—100 people: Breakfast, morning break, lunch, afternoon break, and all day beverage service. (All day beverage service includes water stations, coffee, assorted black and herbal teas, and a variety of soft drinks.)
-		* Main conference, 3 days—450 people per day: Breakfast, morning break, lunch, afternoon break, and all day beverage service. (All day beverage service includes water stations, coffee, assorted black and herbal teas, and a variety of soft drinks.)
-		* Sprints, 2 days—75 people each day: Breakfast, morning break, lunch, afternoon break, and all day beverage service. (All day beverage service includes water stations, coffee, assorted black and herbal teas, and a variety of soft drinks.)
+	* Explanation of catering options available at the venue that can cover:
+		* 450 people per day: Breakfast, morning break, lunch, afternoon break, and all day beverage service.
```
Comment (drewbrew) [inline _posts/2023-06-17-call-for-venue-proposal-2024.md]:
```suggestion
		* 450 people per day during the main conference: Breakfast, morning break, lunch, afternoon break, and all day beverage service.
```

**Review by drewbrew — APPROVED** [review]
I like these changes. Just a few thoughts for clarity

File: _posts/2023-06-17-call-for-venue-proposal-2024.md (hunk 1 above)
Comment (drewbrew) [reply _posts/2023-06-17-call-for-venue-proposal-2024.md:51]: Maybe suggest a local A/V vendor or two if they know of one?

**Pull request merged**